curvefs/src/mds/fs_info_wrapper.cpp

```diff
@@ -75,15 +75,32 @@ FsInfoWrapper::FsInfoWrapper(const ::curvefs::mds::CreateFsRequest* request,
     fsInfo_ = std::move(fsInfo);
 }
 
-bool FsInfoWrapper::IsMountPointExist(const Mountpoint& mp) const {
+bool FsInfoWrapper::IsMountPointExist(const Mountpoint &mp) const {
     return std::find_if(fsInfo_.mountpoints().begin(),
                         fsInfo_.mountpoints().end(),
-                        [mp](const Mountpoint& mountPoint) {
+                        [mp](const Mountpoint &mountPoint) {
                             return mp.path() == mountPoint.path() &&
                                    mp.hostname() == mountPoint.hostname();
                         }) != fsInfo_.mountpoints().end();
 }
 
+bool FsInfoWrapper::IsMountPointConflict(const Mountpoint &mp) const {
+    bool cto = (fsInfo_.mountpoints_size() ? false : mp.cto());
```

Comment:
what does this cto mean?
What is the relationship between mountpoints_size and cto

Reply:
On a mount point, there are only two ways to mount multiple clients. One is that all mount points are all enableCto and the other is that all mount points are not enableCto.

The cto variable here is to record, whether the current mount point of the file system has cto enabled. For the initialization of this variable, if the file system already has a mount point, set it to false, and assign it later when traversing; if the file system has no mount point, set it to be the same as the current mount point.